app/views/layouts/application.html.haml

```diff
@@ -15,6 +15,8 @@
     = render "head"
     = translation_missing_warnings
 
+    %link{rel: "manifest", href: "/manifest.json"}
```

Comment:
This can be added to [_head.haml](https://github.com/diaspora/diaspora/blob/develop/app/views/application/_head.haml) which is used for both, desktop and mobile.

Also, can/should we remove the old `mobile-web-app-capable` now?

https://github.com/diaspora/diaspora/blob/d6225daaf3503f5935133e0e9f109c7bee485e6d/app/views/application/_head.haml#L7